scripts/mocks.mockgen.txt

```diff
@@ -12,16 +12,13 @@ github.com/ava-labs/avalanchego/snow/engine/avalanche/vertex=LinearizableVM=snow
 github.com/ava-labs/avalanchego/snow/engine/snowman/block=BuildBlockWithContextChainVM=snow/engine/snowman/block/mocks/build_block_with_context_vm.go
 github.com/ava-labs/avalanchego/snow/engine/snowman/block=ChainVM=snow/engine/snowman/block/mocks/chain_vm.go
 github.com/ava-labs/avalanchego/snow/engine/snowman/block=StateSyncableVM=snow/engine/snowman/block/mocks/state_syncable_vm.go
-github.com/ava-labs/avalanchego/snow/engine/snowman=Engine=snow/engine/snowman/mock_engine.go
```

Comment:
All this changes are just reverting diff that we have with avax. Probably was lost in previous merges.